tests/jest/unit/modules/playing-cards/assets/js/frontend/handlers/playing-cards-handler.test.js

```diff
@@ -0,0 +1,88 @@
+describe( 'PlayingCardsHandler', () => {
```

Comment:
On tests we are usually adding comments of // Arrange. // Act. // Assert for convenience.